.devcontainer/devcontainer.json

```diff
@@ -1,10 +1,46 @@
 // For format details, see https://aka.ms/devcontainer.json. For config options, see the
 // README at: https://github.com/devcontainers/templates/tree/main/src/docker-existing-dockerfile
 {
-	"name": "Node",
-	"image": "node:16.20.2-alpine3.18",
-	"onCreateCommand": "apk add --no-cache git openssh",
-	"remoteUser": "root"
+	"name": "Existing Dockerfile",
+	// image with NOT included Cypress is used becuase cypress/included doesn't support browsers installatin for arm64 yet
+	"image": "cypress/browsers:node-20.11.1-chrome-123.0.6312.58-1-ff-124.0-edge-122.0.2365.92-1",
+	"features": {
+		"ghcr.io/devcontainers/features/docker-outside-of-docker:1.4.5": {
```

Comment:
get rid of not needed deps, there is no k8s, and some other things are redundant for this case